go/src/meguca/geoip/db.go

```diff
@@ -59,3 +83,137 @@ func LookUp(ip string) (iso string) {
 
 	return
 }
+
+// check if the GeoLite2 country DB exists, and download if it doesn't
+func check() error {
+	// Get the MD5 hash from the DB and the new hash from upstream
+	oldHash, err := db.GetGeoMD5()
+
+	if err != nil {
+		return err
+	}
+
+	respMD5, err := http.Get("https://geolite.maxmind.com/download/geoip/database/GeoLite2-Country.tar.gz.md5")
+
+	if err != nil {
+		return err
+	}
+
+	defer respMD5.Body.Close()
+	md5Bytes, err := ioutil.ReadAll(respMD5.Body)
+
+	if err != nil {
+		return err
+	}
+
+	newHash := string(md5Bytes)
+	diff := !(oldHash == newHash)
```

Comment:
funny

Reply:
I figured it would be better than calculating twice, am I wrong?

Reply:
I meant not using !=

Reply:
I feel stupid now.
I did that when I was tired, my bad.